### PR TITLE
APPS-670 Add to documentation about what you'll see in project & applet job scripts when compiling.

### PR DESCRIPTION
**Pull request by r-i-v-a**

### CODE REVIEW

**Comment by r-i-v-a** [comment]
@jdidion @jputerova   Some documentation improvements for APPS-670

**Review by jdidion — APPROVED** [review]

File: doc/Internals.md [hunk 1]
```diff
@@ -29,13 +29,32 @@ A WDL workflow is compiled into an equivalent DNAnexus workflow, enabling runnin
...
 expensive workflow to fail. 
 * We want to minimize the context that needs to be kept around for the WDL workflow, because it limits job manager scalability.
 
+### Examining compilation output
+
+After compiling your workflow to a project, you will see the compiled workflows and applets in
+the project like this:
+
+```
+my_task_1
```
Comment (emiloslavsky) [inline doc/Internals.md:45]: are there any hidden applets? frags?

Comment (r-i-v-a) [reply doc/Internals.md:45]: The example reflects the visible applets according to the current implementation, so not anything that would be compiled as hidden.

Comment (r-i-v-a) [reply doc/Internals.md:45]: @emiloslavsky   I left myself a note on https://jira.internal.dnanexus.com/browse/APPS-623 to revisit this doc while implementing the bundledDepends task, and ensure the explanation about visibility is improved / up to date then.

**Pull request merged**